### PR TITLE
[React Native]: export thoughts

### PR DESCRIPTION
**Pull request by brunokiafuka**
Added the functionality that will allow users to export/share their thoughts and/or also copy them to the clipboard.

### CODE REVIEW

File: src/components/ModalExport.native.tsx [hunk 1]
```diff
@@ -56,18 +126,16 @@ const ModalExport = () => {
...
-  const [exportContent] = useState<string | null>(null)
+  const [selected, setSelected] = useState(exportOptions[1])
+  const [selectedOptionIndex, setSelectedOptionIndex] = useState(0)
+  const [exportContent, setExportContent] = useState<string | null>(null)
   const [shouldIncludeMetaAttributes, setShouldIncludeMetaAttributes] = useState(true)
   const [shouldIncludeArchived, setShouldIncludeArchived] = useState(true)
-  // const [numDescendantsInState, setNumDescendantsInState] = useState<number | null>(null)
-  const numDescendantsInState = 0
+  const [numDescendantsInState, setNumDescendantsInState] = useState<number | null>(null)
+  // const numDescendantsInState = 0
```
Comment (Foosballfan) [inline src/components/ModalExport.native.tsx:136]: a few unuseful comments here

File: src/selectors/exportContext.ts [hunk 2]
```diff
@@ -15,6 +16,8 @@ const replaceTitle = (text: string, title: string, format: MimeType) => {
 
 /** Strips out HTML tags when exporting as plain text. */
 const stripHTMLTag = (str: string) => {
+  if (isMobile()) return str
+
```
Comment (shresthabijay) [inline src/selectors/exportContext.ts:20]: You need to strip out the HTML tags here. Can you see if this library works [`html-to-text`] in native (https://www.npmjs.com/package/html-to-text) ?

File: src/components/ModalExport.native.tsx [hunk 3]
```diff
@@ -360,4 +402,21 @@ const styles = StyleSheet.create({
   },
 })
 
-export default ModalExport
```
Comment (shresthabijay) [inline src/components/ModalExport.native.tsx:363]: https://github.com/cybersemics/em/blob/05142c9b35f60f67b10366f9fe2bd1b6f1fbc185/src/components/ModalExport.tsx#L137-L150

You have missed adding this component. `ExportThoughtsPhrase` updates the number of descendants in real time when they are being pulled. 

https://github.com/cybersemics/em/blob/05142c9b35f60f67b10366f9fe2bd1b6f1fbc185/src/components/ModalExport.tsx#L418

**Review by shresthabijay — CHANGES_REQUESTED** [review]

File: src/selectors/exportContext.ts (hunk 2 above)
Comment (brunokiafuka) [reply src/selectors/exportContext.ts:20]: Used regex instead.
`return str.replace(/(<([^>]+)>)/gi, '')`

**Review by shresthabijay — APPROVED** [review]

**Pull request merged**